tests/installation/test_executor.py

```diff
@@ -356,8 +356,14 @@ def verify_installed_distribution(venv, package, url_reference=None):
 
     direct_url_file = distribution._path.joinpath("direct_url.json")
 
+    record_file = distribution._path.joinpath("RECORD")
+    with open(record_file, "r") as f:
+        record_file_contents = f.read().splitlines()
+    direct_url_entry = direct_url_file.relative_to(record_file.parent.parent)
```

Comment:
This might be better off as a csv reader populating a set from the first column. Should make the assertion as simple as:

```py
assert direct_url_entry in records
```

Reply:
Alternatively maybe use `Path.read_text` and/or `Path.open`.

Reply:
Hm, not entirely sure what you mean here.

The records are returned as a list of strings. These strings are comma separated. So I can't just `assert direct_url_entry in record_file_contents`.

I believe you mean that my assert was overly complicated with that list comprehension expression, and I agree.

I interpret your comment such as that there should be a nicer/cleaner/simpler way of asserting. I believe we need to assert that we should have at least one row in the `RECORD` file which begins with the direct_url.json file. So I am not sure if `Path.read_text` will cut it.

I'll make a couple of suggestions.

Reply:
Creating another list we can assert with:

```python
    with open(record_file, "r") as f:
        records_first_column = [row.split(",")[0] for row in f]
    assert str(direct_url_entry) in records_first_column
```

Using `Path.read_text`:

```python
assert str(direct_url_entry) in record_file.read_text()
```

@abn do you think the `Path.read_text` approach is sufficient or should I go with the one which checks the first column of the csv RECORD file?

Reply:
I committed the first example to the test, as I think it's better to check only the first column's values.